### PR TITLE
docs: k8s 1.22 and tekton breaking change

### PR DESCRIPTION
**Pull request by babadofar**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# Checklist:

- [ ] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
- [ ] I have signed off the commit, as per instructions mentioned [here](https://jenkins-x.io/community/code/#how-to-sign-your-commits).
- [ ] Any dependent changes have already been merged.

### CODE REVIEW

File: content/en/blog/news/2022-04-22-k8s-122-tekton.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+#### Changes in Tekton version 28
+Tekton made changes in how to calculate the resources needed to run a pipeline, in order to support the concept of [LimitRange](https://kubernetes.io/docs/concepts/policy/limit-range/) in Kubernetes (introduced in Kubernetes version 1.10). Previously, Tekton simply used the maximum requested cpu and memory of any single step, and set that as limits for the all steps in the pipeline. 
+
+[Tekton documentation on how resources are calculated with LimitRange](https://tekton.dev/vault/pipelines-main/limitrange/)
+
+
+#### How to prepare for upgrade
+
+The Jenkins X pipeline catalog has already been upgraded to handle the changes. You can study [this PR](https://github.com/jenkins-x/jx3-pipeline-catalog/pull/984/files) to see how the new resource settings are handled there.
```
Comment (ankitm123) [inline content/en/blog/news/2022-04-22-k8s-122-tekton.md:27]: So we are going to merge the catalog PR before this blog goes live?

**Comment by ankitm123** [comment]
/lgtm
/hold
remove the hold once you are ok to merge this ...

File: content/en/blog/news/2022-04-22-k8s-122-tekton.md (hunk 1 above)
Comment (babadofar) [reply content/en/blog/news/2022-04-22-k8s-122-tekton.md:27]: I guessed that these changes would work in the old version too. I don't really know if that is true yet.

Comment (babadofar) [reply content/en/blog/news/2022-04-22-k8s-122-tekton.md:27]: It worked on my system. But I suppose the consequences of introducing these changes in the old version are uncertain. I will change the text.

File: content/en/v3/admin/platforms/k3s/_index.md [hunk 2]
```diff
@@ -19,8 +19,6 @@ Ensure you are logged into GitHub else you will get a 404 error when clicking th
```
Comment (ankitm123) [inline content/en/v3/admin/platforms/k3s/_index.md:19]: Let's remove these changes from this PR, and publish the blog. We can address the issues with k3s installation in a separate PR, wdyt?

Comment (babadofar) [reply content/en/v3/admin/platforms/k3s/_index.md:19]: Of course. Just me and git not being friends 😝

File: content/en/blog/news/2022-04-22-k8s-122-tekton.md (hunk 1 above)
Comment (ankitm123) [reply content/en/blog/news/2022-04-22-k8s-122-tekton.md]: > But I suppose the consequences of introducing these changes in the old version are uncertain

These changes are backwards compatible afaict

**Pull request merged**